docs/index.rst

```diff
@@ -2,19 +2,47 @@
 ops library API reference
 =========================
 
+The `ops` library is a Python framework for writing and testing Juju charms.
+
+  See more: `Charm SDK documentation <https://juju.is/docs/sdk>`_
+
+The library provides:
+
+- :ref:`ops_main_entry_point`, used to initialise and run your charm;
+- :ref:`ops_module`, the API to respond to Juju events and manage the application;
+- :ref:`ops_pebble_module`, the low-level API for Pebble in Kubernetes containers;
+- :ref:`ops_testing_module`, the framework for unit testing charms in a simulated environment;
+
+You can write a charm in any way you want, but with the `ops` library you get a
+framework that helps you write consistent readable charms following the latest
+best practices, reuse code via charm libs, and separate typical charm concerns
+--- application state management from integration management or lifecycle
+management from testability.
```

Comment:
This seems a bit word salady, and I don't really know what it means. Can we drop it, or make it clearer and less jargony?

Also, nit: usually in typography you use either an m-dash without spaces around it, or an n-dash with spaces around it (not an m-dash with spaces around it as you have here). [See here](https://proofed.com/writing-tips/en-dashes-and-em-dashes-in-us-and-uk-english/).

Reply:
Good catch, I've rewritten the entire paragraph.